source/Nabla/CIrrDeviceStub.cpp

```diff
@@ -50,6 +61,8 @@ CIrrDeviceStub::CIrrDeviceStub(const SIrrlichtCreationParameters& params)
 	// init EGL
 	Display = eglGetDisplay(EGL_DEFAULT_DISPLAY);
 	eglInitialize(Display, NULL, NULL);
+	if (CreationParams.DriverType == video::EDT_OPENGL)
+		eglBindAPI(EGL_OPENGL_API);

```

Comment:
you're supposed to call this in a thread, when creating the context that will be run in that thread
https://www.khronos.org/registry/EGL/sdk/docs/man/html/eglBindAPI.xhtml